Eeems/oxide
# Feature/shutdown feedback

How to test:

1. Install updates packages from build artifacts
2. Reboot
3. Sign in if you have a pin
4. Reboot, you should now get feedback when quitting telling you what is happening, as well as a quick goodbye message
